blockchains/smartchain/assets/0x6Fb05b156788E88c8ad1e057e729362ff8c39d93/info.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Aircoins",
+  "website": "https://www.aircoins.io/",
+  "coinmarketcap": "https://coinmarketcap.com/currencies/aircoins",
+  "coingecko": "https://www.coingecko.com/en/coins/aircoins",
+  "reddit": "https://www.reddit.com/r/Aircoins",
+  "telegram": "https://t.me/aircoins",

```

Comment:
Can you delete unnecessary field? That are not required by our format

Reply:
> Can you delete unnecessary field? That are not required by our format

I am unable to edit, it's saying to delete all file, could you plz give favor to do it?

Reply:
@vikmeup it has been fixed, Please check, thanks
Hope, you also will fetch token value with cmc and cg